syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTContactSection";
option optimize_for = LITE_RUNTIME;

import public "ContactEnums.proto";
import public "ContactItem.proto";

message ContactSection {
    optional uint32 version = 1;
    optional ContactSectionName name = 2;
    repeated ContactItem item = 3;
}
